app/src/main/java/kr/co/connect/boostcamp/livewhere/data/dao/RecentSearchDAO.kt

```diff
@@ -0,0 +1,21 @@
+package kr.co.connect.boostcamp.livewhere.data.dao
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import kr.co.connect.boostcamp.livewhere.data.entity.RecentSearchEntity
+
+
+@Dao
+interface RecentSearchDAO {
```

Comment:
검색에 대한 정렬은 언제든지 가능하므로, 검색 기록이라는 의미로 아래와 같이 바꾸는 것은 어떨까 제안드립니다.
```suggestion
interface SearchHistoryDao {
```